### PR TITLE
test-model-story-comment-chapter-author

### PR DESCRIPTION
**Pull request by tunglam96hd**

### CODE REVIEW

**Comment by tunglam96hd** [comment]
https://edu-redmine.framgia.vn/issues/5301

**Comment by tunglam96hd** [comment]
/home/tung/Pictures/Screenshot from 2019-01-08 16-11-07.png

**Comment by tunglam96hd** [comment]
image

**Comment by tunglam96hd** [comment]
ready

File: spec/models/chapter_spec.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+RSpec.describe Chapter, type: :model do
+  describe "associations" do
+    it { is_expected.to belong_to(:member) }
+    it { is_expected.to belong_to(:story) }
+  end
+
+  describe "db schema" do
+    context "columns" do
+      it { is_expected.to have_db_column(:content).of_type(:text) }
+      it { is_expected.to have_db_column(:content).of_type(:text) }
```
Comment (huongnguyenmta) [inline spec/models/chapter_spec.rb]: duplicate nè cưng

**Comment by huongnguyenmta** [comment]
commented
Factory author và chapter nữa nhé em ;)

**Comment by tunglam96hd** [comment]
image

**Comment by tunglam96hd** [comment]
ready

**Pull request merged**